src/pages/api/csvUpload.ts

```diff
@@ -74,7 +83,12 @@ export const faresTriangleDataMapper = (
     for (let rowNum = 0; rowNum < dataAsLines.length; rowNum += 1) {
         const items = dataAsLines[rowNum].split(',');
         const trimmedItems = items.map(item => item.trim());
-        const stageName = trimmedItems[rowNum + 1];
+        let stageName = '';
+        if (isNotTicketerFormat(dataAsLines)) {
```

Comment:
could we do this in one line

`const stageName = isNotTicketerFormat(dataAsLines) ? trimmedItems[rowNum + 1] : trimmedItems[rowNum];`